dist/types/audiokeys.d.ts

```diff
@@ -0,0 +1,26 @@
+type AudioKeysOptions = {
+  polyphony: number
+  rows: 1 | 2
+  priority: 'last' | 'first' | 'highest' | 'lowest'
+  rootNote: number
+  octaveControls: boolean
+  velocityControls: boolean
+}
+
+interface AudioKeysNoteData {
+  velocity: number // the velocity, between 0 and 127
+  keyCode: number // the keyboard key pressed or released
+  note: number
+  frequency: number
+  isActive: boolean
+}
+
+declare module 'audiokeys' {
+  export default class AudioKeysDefault {
+    constructor(options: Partial<AudioKeysOptions>)
```

Comment:
Thanks for making this file. Does it need a `?` here on `options` in case they aren't passed in?

Reply:
Ah, yes indeed, good catch, thanks!